minio/minio
# Fixes browser delete issue for anon and authorized users

## Description
Fixes issue #9410, and the regression introduced by #9259 (`"fix: support object-remaining-retention-days policy condition"`).

## Motivation and Context
To let authorized users to be able to delete a file.

## How to test this PR?
Please see the reproduction steps explained in issue #9410.
Also repeated the same repro steps for an authorized user with lock/retention enabled and set.

## Types of changes

## Checklist:
- [x] Fixes a regression (#9259)
